components/NurseTeam/NurseTeam.js

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import useSWR from "swr";
+import {
+  StyledListContainer,
+  StyledHeading,
+  StyledList,
+  StyledListItem,
+} from "./NurseTeam.styled";
+
+const fetcher = (url) => fetch(url).then((res) => res.json());
+
+export default function NurseTeam() {
+  const { data, error } = useSWR("/api/nurses", fetcher);
+  const [nurses, setNurses] = useState([]);
+
+  useEffect(() => {
+    if (data) {
+      setNurses(data);
+    }
+  }, [data]);
```

Comment:
It's considered bad practise to create a separate state which stores ALL the data from your fetch. A better way would be to store only the ids. That way you can add additional info to your separate state without duplicating the data.

Reply:
I will write more simple this time, no state! Then in the next ones I will add states when more necessary.